app/views/stats/show.html.haml

```diff
@@ -1,5 +1,6 @@
 = govuk_page_heading(t('stats.show.title'))
-
+%p.govuk-hint
+  = t('stats.show.hint')
```

Comment:
@pliao-hmcts, this is perfectly done. But I think we are overusing the I18.

We are not going to translate this.
Could you inline the hint and remove the entry from the stats.yml?